src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Have never said the certs shouldn't be updated to match a config resotore/swap/move.  Rewriting them all at ever boot to do it is the issue.  Totally unnecessary.  And to require checksuming them and not the CA bundle is illogical.

So the CA bundle could be verified if you wanted it to be.  But for some reason you don't and are willing to trust it not to get corrupted, but not these files.  That's what is not logical.